clingo/README.md

```diff
@@ -12,4 +12,39 @@ Binary package
 
 ## Usage
 
-*TODO: Add instructions for usage*
+Typically this is a runtime dependency that can be added to your
+plan.sh:
+
+    pkg_deps=(core/clingo)
+
+## Testing
+
+From the root directory of core-plans, open a terminal and run the following:
+
+```bash
+# build clingo from its own directory
+( cd clingo && hab pkg build . )
+
+# set environment from the last clingo build
+source clingo/results/last_build.env
+
+# Run the tests
+hab studio run "./clingo/tests/test.sh $pkg_ident"
+```
+
+Sample output:
+
+```bash
+(travis_ci_version_2) ➜  core-plans git:(gavindidrichsen/clingo_fix_v1) . ./clingo/results/last_build.env
+(travis_ci_version_2) ➜  core-plans git:(gavindidrichsen/clingo_fix_v1) hab studio run "./clingo/tests/test.sh $pkg_ident"
+   hab-studio: Importing 'core' secret origin key
+...
+...
+★ Install of core/clingo/5.3.0/20190520203121 complete with 7 new packages installed.
+1..2
+ok 1 package directory for package ident core/clingo/5.3.0/20190520203121 exists
+ok 2 gringo should work as expected
+(travis_ci_version_2) ➜  core-plans git:(gavindidrichsen/clingo_fix_v1)
+```
+
+Note that the test files beneath **./tests/examples/gringo/sort** are from [potassco github](https://github.com/potassco/clingo/tree/master/examples/gringo/sort)
```

Comment:
Just my personal preference, but I think the `examples` directory would be more descriptively named as `fixtures`

Reply:
I believe `examples` is correct here, as it follows the upstream naming/format.

Reply:
Moved 'examples' beneath 'fixtures' so now './tests/fixtures/examples' instead of './tests/examples'